### PR TITLE
Add about us page, homepage, placeholder(image list) page, & updated css.

### PR DESCRIPTION
**Pull request by Corramel**

### CODE REVIEW

**Review by InfinityGrip — APPROVED** [review]

File: .theia/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (matty377) [inline .theia/settings.json:1]: You should add `.theia` to the `.gitignore`. Is there a reason to track this file in the main repository?

File: src/main/webapp/about-us.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		<ul>
+			<li><a href="home.html">Home</a></li>
+			<li class="currentPage"><a href="about-us.html">About Us</a></li>
+			<li><a href="placeholder.html">Images and Messages</a></li>
+			<li><a href="index.html">Image Upload</a></li>
+		</ul>
+	</nav>
+	<div id="container">
+		<p class = "homePageP">
+			Our team: Project Advisor (Matt Simmons), Eddy Nkere, Facundo Munoz, Godbless Chille, and Jontae Greene.
```
Comment (matty377) [inline src/main/webapp/about-us.html:20]: You should put me last or leave me out. You're all far more important.

File: src/main/webapp/about-us.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		</p> <!---Perhaps add links to everyone's portfolio? Could be good.--->
```
Comment (matty377) [inline src/main/webapp/about-us.html:21]: Cool idea!

File: src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,43 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
-#greeting-container {
-  margin-top: 20px;
-}
+/* CSS Document */
```
Comment (matty377) [inline src/main/webapp/style.css:1]: This comment is probably unnecessary.

**Review by matty377 — APPROVED** [review]
LGTM. Added a few suggestions.

**Pull request merged**